entegywrapper/profiles/profiles.py

```diff
@@ -330,7 +330,7 @@ def update_profile(
 def sync_profiles(
     self: EntegyAPI,
     update_reference_type: ProfileIdentifier,
-    profiles: list[Profile],
+    profiles: list[Profile | ProfileUpdate],
```

Comment:
I know it pre-existed this PR, but maybe change `Profile` to `ProfileCreate`:

```suggestion
    profiles: list[ProfileCreate | ProfileUpdate],
```

You shouldn't be able to give this method a full `Profile` object.

Reply:
It looks like `ProfileCreate` is very wrong, will need to update it

Reply:
Oh shit, it is *very* wrong.